src/components/timeline/Lane.vue

```diff
@@ -63,12 +64,20 @@ export default {
     };
   },
   methods: {
+    onMouseOver() {
+      console.log("onMouseOver " + this.assignee.id);
+      sessionStorage.setItem("assigneeID", this.assignee.id);
```

Comment:
As I already told you, I don't really like the use of sessionStorage here. Setting something globally shouldn't really be necessary in this situation. (And if so, our vuex store would be the better place for that.)
I'm especially concerned, because you're setting something there on every mouse over event, not only when the user is actually dragging a task. This seems to be a huge performance issue.

I will be trying to find another solution tomorrow, where the communication between the task and the "new lane" is more direct.